src/python/dxpy/cli/exec_io.py

```diff
@@ -31,7 +31,8 @@
 from ..utils.printing import (RED, GREEN, WHITE, BOLD, ENDC, UNDERLINE, fill)
 from ..utils.describe import (get_find_executions_string, get_ls_l_desc, get_ls_l_desc_fields, parse_typespec)
 from ..utils.resolver import (parse_input_keyval, is_hashid, is_job_id, is_localjob_id, paginate_and_pick, pick,
-                              resolve_existing_path, resolve_multiple_existing_paths, split_unescaped, is_analysis_id)
+                              resolve_existing_path, resolve_multiple_existing_paths, split_unescaped, is_analysis_id,
+                              parse_obj)
```

Comment:
I'm wondering if it's ok to change the location of this method, as it's not backwards compatible (some of our customers may be using this method).

Reply:
I've put it back to its place. Though it seems that resolver is a better place to have this method in.